resources/web/wwi/FloatingProtoParameterWindow.js

```diff
@@ -496,9 +500,8 @@ export default class FloatingProtoParameterWindow extends FloatingWindow {
     // Separately printing only keys
     const array = [];
     let i = 0;
-    for (let value of lut.values()) {
+    for (let value of lut.values())
```

Comment:
we should probably rename this `lut`, since it might be things other than look up tables